token_contracts/baln/balance.py

```diff
@@ -121,6 +121,10 @@ def on_install(self, _governance: Address) -> None:
 
     def on_update(self) -> None:
         super().on_update()
+        old_div_address = Address.from_string('cx13f08df7106ae462c8358066e6d47bb68d995b6d')
+        new_div_address = Address.from_string('cx203d9cd2a669be67177e997b8948ce2c35caffae')
+        old_div_balance = self._balances[old_div_address]
+        self._transfer(old_div_address, new_div_address, old_div_balance)
```

Comment:
You need to specify the `_data` param as follows, otherwise it will raise param mismatch exception.
```suggestion
        self._transfer(old_div_address, new_div_address, old_div_balance, b'')
```

Reply:
Ah, of course. Thanks!